crates/base/src/deno_runtime.rs

```diff
@@ -207,8 +207,8 @@ impl EdgeRuntime {
                 }
             },
             get_error_class_fn: Some(&get_error_class_name),
-            shared_array_buffer_store: None,
-            compiled_wasm_module_store: None,
+            shared_array_buffer_store: Default::default(),
```

Comment:
Do we need to set them?

Reply:
Will enable only the `compiled_wasm_module_store`. It should speed up wasm usages across isolates.